### PR TITLE
Import mongodb package

### PR DESCRIPTION
**Pull request by kaiyan-sheng**
This PR is to migrate `mongodb` into integrations:
Screen Shot 2020-06-29 at 3 55 08 PM (2)

add mongodb datasets:
Screen Shot 2020-06-29 at 3 07 18 PM (2)

mongodb logs dashboard:
Screen Shot 2020-06-29 at 2 40 12 PM

mongodb metrics dashboard:
Screen Shot 2020-06-29 at 2 58 06 PM (2)

### CODE REVIEW

File: dev/import-beats-resources/mongodb/docs/README.md [hunk 1]
```diff
@@ -0,0 +1,138 @@
+# MongoDB Integration
+
+This integration is used to fetches logs and metrics from
```
Comment (mtojek) [inline dev/import-beats-resources/mongodb/docs/README.md:3]: nit: to fetch

File: dev/import-beats-resources/mongodb/docs/README.md [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+used and a count of operations for the following types: total, readLock, writeLock,
+queries, getmore, insert, update, remove, and commands.
+
+It requires the following privileges, which is covered by the [clusterMonitor](https://docs.mongodb.com/manual/reference/built-in-roles/#clusterMonitor) role:
+
+* [top action](https://docs.mongodb.com/manual/reference/privilege-actions/#top) on [cluster resource](https://docs.mongodb.com/manual/reference/resource-document/#cluster-resource)
+
+An example event for `collstats` looks as following:
+
+```$json
```
Comment (mtojek) [inline dev/import-beats-resources/mongodb/docs/README.md:63]: missing sample event

File: dev/import-beats-resources/mongodb/docs/README.md [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+
+* [listDatabases](https://docs.mongodb.com/manual/reference/privilege-actions/#listDatabases) 
+action on [cluster resource](https://docs.mongodb.com/manual/reference/resource-document/#cluster-resource)
+
+* for each of the databases, also need [dbStats](https://docs.mongodb.com/manual/reference/privilege-actions/#dbStats)
+action on the [database resource](https://docs.mongodb.com/manual/reference/resource-document/#database-and-or-collection-resource)
+
+An example event for `dbstats` looks as following:
+
+```$json
```
Comment (mtojek) [inline dev/import-beats-resources/mongodb/docs/README.md:84]: missing sample event

File: dev/import-beats-resources/mongodb/docs/README.md [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+The `replstatus` dataset collects status of the replica set.
+It requires the following privileges, which is covered by the [clusterMonitor](https://docs.mongodb.com/manual/reference/built-in-roles/#clusterMonitor) role:
+
+* [find/listCollections](https://docs.mongodb.com/manual/reference/privilege-actions/#find) action on the [local database](https://docs.mongodb.com/manual/reference/local-database/) resource
+* [collStats](https://docs.mongodb.com/manual/reference/privilege-actions/#collStats) action on the [local.oplog.rs](https://docs.mongodb.com/manual/reference/local-database/#local.oplog.rs) collection resource
+* [replSetGetStatus](https://docs.mongodb.com/manual/reference/privilege-actions/#replSetGetStatus) action on [cluster resource](https://docs.mongodb.com/manual/reference/resource-document/#cluster-resource)
+
+An example event for `replstatus` looks as following:
+
+```$json
```
Comment (mtojek) [inline dev/import-beats-resources/mongodb/docs/README.md:117]: same here

File: dev/import-beats-resources/mongodb/docs/README.md [hunk 5]
```diff
@@ -0,0 +1,138 @@
...
+### metrics
+
+It requires the following privileges, which is covered by the clusterMonitor role:
+
+* [serverStatus](https://docs.mongodb.com/manual/reference/privilege-actions/#serverStatus) 
+action on [cluster resource](https://docs.mongodb.com/manual/reference/resource-document/#cluster-resource)
+
+An example event for `metrics` looks as following:
+
+```$json
```
Comment (mtojek) [inline dev/import-beats-resources/mongodb/docs/README.md:100]: same here

File: dev/import-beats-resources/mongodb/docs/README.md [hunk 6]
```diff
@@ -0,0 +1,138 @@
...
+### status
+
+It requires the following privileges, which is covered by the [clusterMonitor](https://docs.mongodb.com/manual/reference/built-in-roles/#clusterMonitor) role:
+
+* [serverStatus](https://docs.mongodb.com/manual/reference/privilege-actions/#serverStatus) 
+action on [cluster resource](https://docs.mongodb.com/manual/reference/resource-document/#cluster-resource)
+
+An example event for `status` looks as following:
+
+```$json
```
Comment (mtojek) [inline dev/import-beats-resources/mongodb/docs/README.md:133]: same here

File: dev/import-beats-resources/mongodb/docs/README.md (hunk 2 above)
Comment (kaiyan-sheng) [reply dev/import-beats-resources/mongodb/docs/README.md]: done

File: dev/import-beats-resources/mongodb/docs/README.md (hunk 4 above)
Comment (kaiyan-sheng) [reply dev/import-beats-resources/mongodb/docs/README.md]: done

File: dev/import-beats-resources/mongodb/docs/README.md (hunk 5 above)
Comment (kaiyan-sheng) [reply dev/import-beats-resources/mongodb/docs/README.md]: done

File: dev/import-beats-resources/mongodb/docs/README.md (hunk 6 above)
Comment (kaiyan-sheng) [reply dev/import-beats-resources/mongodb/docs/README.md]: done

File: dev/import-beats-resources/mongodb/docs/README.md (hunk 3 above)
Comment (kaiyan-sheng) [reply dev/import-beats-resources/mongodb/docs/README.md]: done

**Pull request merged**